Eevee.Sleep.Bot/Utils/DiscordMessageMaker/DiscordMessageMakerForRoleRestriction.cs

```diff
@@ -0,0 +1,68 @@
+using Discord;
+using Eevee.Sleep.Bot.Enums;
+using Eevee.Sleep.Bot.Extensions;
+
+namespace Eevee.Sleep.Bot.Utils.DiscordMessageMaker;
+
+public static class DiscordMessageMakerForRoleRestriction {
+    public static Embed MakeRoleRestrictedNote(
+        string roleName,
+        IUser user,
+        uint accountAgeDays,
+        string guildName
+    ) {
+        var serverlink = $"[{guildName}](https://discord.com/channels/{ConfigHelper.GetDiscordWorkingGuild()})";
+        var dueTo = user.CreatedAt.ToUniversalTime().AddDays(accountAgeDays);
```

Comment:
Should be something literally `<t:1726935360:R>` with the middle part being the epoch seconds.